### PR TITLE
Added Tarsier as an entry in the bot facts

### PR DESCRIPTION
**Pull request by melvincanafranca**
Hi! Thanks for the opportunity to collaborate with you in creating this animal bot. I have added Tarsier as one of the animal entry in the facts list. Kindly let me know if you have concerns.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -174,6 +174,7 @@ You can see the bot in action at https://www.reddit.com/user/AnimalFactsBot/comm
 * Tarantula
 * Tardigrade
 * Tasmanian devil
+* Tarsier
```
Comment (mpentler) [inline README.md]: Tarsier belongs before Tasmanian Devil, minor point though, that :)

File: animalfacts.py [hunk 2]
```diff
@@ -2739,6 +2740,16 @@ def animalfactsbot(reddit):
...
     )
+TARSIER_FACTS = (
+    'A Tarsier is one of the known smallest primates. It is an ancient ancestor of a modern day monkey.',
+    'A Tarsier does not grow bigger than a man''s hand. It is about 15cm in height and between 115 and 130g in weight.',
+    'A Tarsier have gray fur and a nearly naked tail.',
+    'A Tarsier is recognizable due to its huge eyes and its long feet. It has the largest eyes, proportionate to its body size, of any animal on the planet.',
+    'Their large eyes enables them to have acute night vision that makes them a good hunter.',
+    'Their long feet enables them to jump from tree to tree up to 3 meters in distance.',
+    'A Tarsier is mostly active at night and lives on a diet of insects.',
+    'A Tarsier can usually be found in the beautiful archipelago called the Philippines.'
```
Comment (mpentler) [inline animalfacts.py]: Could you pop a comma on the end of this line to match the other tuples? Otherwise these facts are all great!

**Comment by mpentler** [comment]
Actually I just realised that all of the facts need to read ok independently of the others. Can you have a look at that and then I'll take em. I added the comma for you

File: animalfacts.py (hunk 2 above)
Comment (melvincanafranca) [reply animalfacts.py]: Thanks for your edits. I'm totally fine with it.

**Comment by mpentler** [comment]
I’ll try and tidy this up for pulling myself tomorrow. Thanks for your work!

File: animalfacts.py [hunk 3]
```diff
@@ -356,6 +356,7 @@ def check_comment_for_animal(comment, reddit):
     botengine('stingray', '\sstingrays?\s', reddit, STINGRAY_FACTS, comment)
     botengine('sunfish', '\ssunfish(es)?\s', reddit, SUNFISH_FACTS, comment)
     botengine('tardigrade', '\stardigrades?\s', reddit, TARDIGRADE_FACTS, comment)
+    botengine('tarsier', '\starsier?\s', reddit, TARSIER_FACTS, comment)
```
[the remark this answers is not in the document]
Comment (mpentler) [reply animalfacts.py]: Fixed it myself

**Comment by mpentler** [comment]
Sorted this out. Ready for pull!

**Pull request merged**